db/migrate/20160720151217_create_arbitration_rules.rb

```diff
@@ -0,0 +1,16 @@
+class CreateArbitrationRules < ActiveRecord::Migration[5.0]
+  def change
+    create_table :arbitration_rules do |t|
+      t.string :type
+      t.string :name
+      t.string :parameter
+      t.string :condition
+      t.string :value
+      t.string :action
+      t.integer :profile_id
```

Comment:
Prefer arbitration_profile_id since that is the foreign key, and then you don't need to manually specify it in the model.